example/80-secretbinding-provider.yaml

```diff
@@ -0,0 +1,13 @@
+# SecretBindings bind a secret from the same or another namespace together with Quotas from the same or other namespaces.
+---
+apiVersion: garden.sapcloud.io/v1beta1
+kind: SecretBinding
+metadata:
+  name: my-provider-account
+  namespace: garden-dev
```

Comment:
What happens with the previously used label `cloudprofile.garden.sapcloud.io/name`?

Reply:
It's not contract of Gardener API - the dashboard uses it internally, so I removed it from our examples.